Mediapipe.Net/Util/UnsafeUtil.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) homuler and Vignette
+// This file is part of MediaPipe.NET.
+// MediaPipe.NET is licensed under the MIT License. See LICENSE for details.
+
+namespace Mediapipe.Net.Util
+{
+    internal unsafe static class UnsafeUtil
+    {
+        public static T[] SafeArrayCopy<T>(T* ptr, int length)
```

Comment:
you couldn't choose a better name

Reply:
Yeah... xD
It copies all the objects from an unsafe pointer to a safe array, so unironically that's the best name I could find lol

Reply:
I think it looks fine, just document that static properly.